ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/UnclosedQuoteCheckerTest.java

```diff
@@ -0,0 +1,62 @@
+package io.confluent.ksql.cli.console;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UnclosedQuoteCheckerTest {
+
+  @Test
+  public void shouldFindUnclosedQuote() {
+    // Given:
+    final String line = "some line 'this is in a comment";
```

Comment:
The string is a bit misleading :P These are not in comments, right?

Reply:
Hah, copied a bit too many times.  I changed them to make sense, I believe.